.github/workflows/workflow.yml

```diff
@@ -0,0 +1,27 @@
+# from https://github.com/anothrNick/github-tag-action, works for merge requests to main
+name: Bump version
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
+
+jobs:
+  build:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-22.04
+    permissions:
+      contents: write
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        ref: ${{ github.event.pull_request.merge_commit_sha }}
+        fetch-depth: '0'
+
+    - name: Bump version and push tag
+      uses: anothrNick/github-tag-action@v1 # Don't use @master or @v1 unless you're happy to test the latest version
```

Comment:
Are you sure this action is releasing the library to a Python package registry (e.g. PyPi)? From what I read on https://github.com/anothrNick/github-tag-action this only tags and does a pre-release

Reply:
You can have a look at this workflow in `lib-ml`: https://github.com/remla2024-team14/lib-ml/blob/main/.github/workflows/release.yml

Reply:
This is an example of pushing to GHs _container_ registry: https://github.com/remla2024-team14/model-service/blob/build-release-and-version/.github/workflows/release.yml

Reply:
It only does tags, it doesn't release to pypi, but I'll make it release to pypi